### PR TITLE
docs: Add 2018/05/10 and 2018/05/24 meeting minutes.

### LINKED ISSUE #40: Node.js Foundation Website Redesign Strategic Initiative Meeting 2018-05-10
## Time

**UTC Thu 10-May-2018 14:00 (02:00 PM)**:

| Timezone      | Date/Time             |
|---------------|-----------------------|
| US / Pacific  | Thu 10-May-2018 07:00 (07:00 AM) |
| US / Mountain | Thu 10-May-2018 08:00 (08:00 AM) |
| US / Central  | Thu 10-May-2018 09:00 (09:00 AM) |
| US / Eastern  | Thu 10-May-2018 10:00 (10:00 AM) |
| London        | Thu 10-May-2018 15:00 (03:00 PM) |
| Amsterdam     | Thu 10-May-2018 16:00 (04:00 PM) |
| Moscow        | Thu 10-May-2018 17:00 (05:00 PM) |
| Chennai       | Thu 10-May-2018 19:30 (07:30 PM) |
| Hangzhou      | Thu 10-May-2018 22:00 (10:00 PM) |
| Tokyo         | Thu 10-May-2018 23:00 (11:00 PM) |
| Sydney        | Fri 11-May-2018 00:00 (12:00 AM) |

Or in your local time:
* http://www.timeanddate.com/worldclock/fixedtime.html?msg=Node.js+Foundation+Website%20Redesign%20Strategic%20Initiative+Meeting+2018-05-10&iso=20180510T14
* or http://www.wolframalpha.com/input/?i=02PM+UTC%2C+May+10%2C+2018+in+local+time

## Links

* Minutes Google Doc: <https://docs.google.com/document/d/1nBFsEmAdSAhqBoHXCpC8qAHbXLTJGieGdpBi2M-msyI/edit>

## Agenda

Extracted from **wr-agenda** labelled issues and pull requests from the **nodejs org** prior to the meeting.

### nodejs/website-redesign

* Website Wireframes [#15](https://github.com/nodejs/website-redesign/issues/15)
* Tech Exploration for In-Browser Node Playground [#12](https://github.com/nodejs/website-redesign/issues/12)
* Extended Color Palette [#42](https://github.com/nodejs/website-redesign/issues/42)
* Identify topics to discuss for Collab Summit Spring 2018 [#29](https://github.com/nodejs/website-redesign/issues/29)
* Discuss Meeting Time

## Invited

* Website Redesign Strategic Initiative Team: @nodejs/website-redesign

## Notes

The agenda comes from issues labelled with `wr-agenda` across **all of the repositories in the nodejs org**. Please label any additional issues that should be on the agenda before the meeting starts.

## Joining the meeting

* link for participants: Will be added a few minutes before meeting starts
* For those who just want to watch:
* youtube admin page: https://www.youtube.com/my_live_events?filter=scheduled

### LINKED ISSUE #50: Node.js Foundation Website Redesign Strategic Initiative Meeting 2018-05-24
## Time

**UTC Thu 24-May-2018 14:00 (02:00 PM)**:

| Timezone      | Date/Time             |
|---------------|-----------------------|
| US / Pacific  | Thu 24-May-2018 07:00 (07:00 AM) |
| US / Mountain | Thu 24-May-2018 08:00 (08:00 AM) |
| US / Central  | Thu 24-May-2018 09:00 (09:00 AM) |
| US / Eastern  | Thu 24-May-2018 10:00 (10:00 AM) |
| London        | Thu 24-May-2018 15:00 (03:00 PM) |
| Amsterdam     | Thu 24-May-2018 16:00 (04:00 PM) |
| Moscow        | Thu 24-May-2018 17:00 (05:00 PM) |
| Chennai       | Thu 24-May-2018 19:30 (07:30 PM) |
| Hangzhou      | Thu 24-May-2018 22:00 (10:00 PM) |
| Tokyo         | Thu 24-May-2018 23:00 (11:00 PM) |
| Sydney        | Fri 25-May-2018 00:00 (12:00 AM) |

Or in your local time:
* http://www.timeanddate.com/worldclock/fixedtime.html?msg=Node.js+Foundation+Website%20Redesign%20Strategic%20Initiative+Meeting+2018-05-24&iso=20180524T14
* or http://www.wolframalpha.com/input/?i=02PM+UTC%2C+May+24%2C+2018+in+local+time

## Links

* Minutes Google Doc: <https://docs.google.com/document/d/1gcQExgTMVkDx440IvfQOmQ9hLxMcd7Y2mCfj-Jp270s/edit>

## Agenda

Extracted from **wr-agenda** labelled issues and pull requests from the **nodejs org** prior to the meeting.

### nodejs/website-redesign

* Node.js Extended Color Palette  [#42](https://github.com/nodejs/website-redesign/issues/42)
* Identify topics to discuss for Collab Summit Spring 2018 [#29](https://github.com/nodejs/website-redesign/issues/29)

## Invited

* Website Redesign Strategic Initiative Team: @nodejs/website-redesign

## Notes

The agenda comes from issues labelled with `wr-agenda` across **all of the repositories in the nodejs org**. Please label any additional issues that should be on the agenda before the meeting starts.

## Joining the meeting

* link for participants: Will be added a few minutes before meeting starts
* For those who just want to watch:
* youtube admin page: https://www.youtube.com/my_live_events?filter=scheduled

### PR DESCRIPTION
**Pull request by amiller-gh**
- Closes #40.
 - Closes #50.

### CODE REVIEW

**Review by sagirk — APPROVED** [review]
LGTM 🎉

**Comment by sagirk** [comment]
@chowdhurian @amiller-gh @maddhruv 
Sorry, perhaps not the right place to post, but hey! How do I get involved?

Is there some criteria/process? Or do I just PR myself in to the README?

File: meetings/2018-05-10.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
+# Node.js Foundation Website Redesign Strategic Initiative Meeting 2018-05-10
+
+## Links
+
+* **Recording**:
```
Comment (Belar) [inline meetings/2018-05-10.md]: https://youtu.be/CnccqFY2HFw

File: meetings/2018-05-24.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
+# Node.js Foundation Website Redesign Strategic Initiative Meeting 2018-05-24
+
+## Links
+
+* **Recording**:
```
Comment (Belar) [inline meetings/2018-05-24.md]: https://youtu.be/2gqCrBBiavs

**Review by Belar — COMMENTED** [review]
Added comments with links to recordings from both meetings

**Review by keywordnew — APPROVED** [review]
LGTM.

Made small updates to phase titles to match the Collab Summit session guide.

**Comment by keywordnew** [comment]
@sagirk That's right! The best way to let the project know that you would like to get involved is to PR yourself into the README.md under the phase in which you'd like to start getting involved. That way, you'll be pinged when it gets to that point.

If you'd like to start right now, we're in the Design phase. The open issues will give you an idea of what's being worked on right now. You're welcome and encouraged to join the meetings that happen online every other week! 💯

**Comment by keywordnew** [comment]
@Belar thank you for adding the links! 👍🏽Your contributions are appreciated!

**Pull request merged**